src/test/java/com/crud/tasks/controller/TaskControllerTest.java

```diff
@@ -38,15 +36,12 @@
     @MockBean
     private TaskMapper taskMapper;
 
-    @Rule
-    public final ExpectedException exception = ExpectedException.none();
-
     private static final long ID = 1L;
     private static final String TITLE = "TestDto";
     private static final String CONTENT = "Test_ContentDto";
-    private Task testeeTask = new Task(1L, "Test", "Test_Content");
-    private TaskDto testeeTaskDto = new TaskDto(ID, TITLE, CONTENT);
-    private List<TaskDto> testeeTaskDtos = Arrays.asList(testeeTaskDto);
+    private static final Task testeeTask = new Task(1L, "Test", "Test_Content");
```

Comment:
Nie uwzględniłeś uwagi

Reply:
Czy jeśli mam jakieś stałe np. private static final long ID = 1L;
to je też powinienem inicjalizować w metodzie @Before?